src/packcc.c

```diff
@@ -282,7 +282,9 @@ static int print_error(const char *format, ...) __attribute__((format(printf, 1,
     n = fprintf(stderr, "%s: ", g_cmdname);
     if (n >= 0) {
         const int k = vfprintf(stderr, format, a);
-        if (k < 0) n = k; else n += k;
+        if (k < 0) n = k;
+        else
+            n += k;
```

Comment:
This is one of the weird things that clang-format does... It can keep short `if` on single line, but I couldn't figure out how to do the same for `else`. I'll have to search more...